.lintrunner.toml

```diff
@@ -900,8 +901,6 @@ exclude_patterns = [
     'test/distributed/test_collective_utils.py',
     'test/distributions/test_distributions.py',
     'test/dynamo/test_compile.py',
-    'test/dynamo/test_sources.py',
-    'test/functorch/test_logging.py',
```

Comment:
What does removing these here mean?

Reply:
I added a lint rule to make sure that test files would have the run_tests call so that people would know to add it when they make a new file.  This is the exclusion list containing preexisting tests that didnt have the call